packages/dbml-parse/package.json

```diff
@@ -22,7 +23,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "vite build",
+    "build": "tsc --noEmit && vite build",
```

Comment:
You added a package to generate the definition file, and does it have type checking?

If yes, why do you have to add `tsc --noEmit` here?

Reply:
It's a common idiom anh :v In this section here https://vitejs.dev/guide/features#transpile-only 

![image](https://github.com/user-attachments/assets/dc82c9cf-79ab-466c-b918-5cf56b862aab)

Reply:
By the way, the package `vite-dts` is a build-only package to gen `.d.ts` file :V this would help providing autocompletion when developing other packages that rely on `@dbml/parse` (the language service would lookup the `.d.ts` files to provide type checking).

Reply:
So because vite only emit transpile the code to `d.ts files`, it doesn't perform type-checking during `vite build`. That's why you add `tsc --noEmit` to type-check beforehand?

Reply:
ye anh, vite only builds anh, so tsc --noEmit is used to perform typecheck before build.